src/odemis/gui/cont/tabs.py

```diff
@@ -1706,6 +1706,101 @@ def get_display_priority(cls, main_data):
+        #   Takes care of the acquisition and acquisition selection buttons.
+
+        tab_data = guimod.FastEMOverviewGUIData(main_data)
+
+        super(FastEMOverviewTab, self).__init__(name, button, panel, main_frame, tab_data)
+        self.set_label("OVERVIEW")
+
+        # View Controller
+        vp = panel.vp_fastem_overview
+        assert(isinstance(vp, FastEMOverviewViewport))
+        vpv = collections.OrderedDict([
+            (vp,
+             {"name": "Overview",
+              "stage": main_data.stage,
+              "cls": guimod.StreamView,
+              "stream_classes": EMStream,
+              "zPos": self.tab_data_model.zPos,
+              }),
+        ])
+        self.view_controller = viewcont.ViewPortController(tab_data, panel, vpv)
+
+        # Single-beam SEM stream
+        vanames = ("resolution", "dwellTime", "horizontalFoV", "pixelSize")
+        main_data.hw_settings_config["e-beam"]["pixelSize"]["control_type"] = odemis.gui.CONTROL_READONLY
+        hwemtvas = set()
+        for vaname in getVAs(main_data.ebeam):
+            if vaname in vanames:
+                hwemtvas.add(vaname)
+        sem_stream = acqstream.SEMStream(
+            "Single Beam",
+            main_data.sed,
+            main_data.sed.data,
+            main_data.ebeam,
+            focuser=main_data.ebeam_focus,
+            hwemtvas=hwemtvas,
+        )
+        tab_data.streams.value.append(sem_stream)  # it should also be saved
+        tab_data.semStream = sem_stream
+        self._stream_controller = streamcont.StreamBarController(
+            tab_data,
+            panel.pnl_fastem_overview_streams,
+            ignore_view=True,  # Show all stream panels, independent of any selected viewport
+            view_ctrl=self.view_controller,
+        )
+        sem_stream_cont = self._stream_controller.addStream(sem_stream, add_to_view=True)
+        sem_stream_cont.stream_panel.show_remove_btn(False)
+
+        # Acquisition controller
+        self._acquisition_controller = acqcont.FastEMOverviewAcquiController(
+            tab_data,
+            panel,
+        )
+        main_data.is_acquiring.subscribe(self.on_acquisition)
+
+    @property
+    def streambar_controller(self):
+        return self._stream_controller
+
+    @property
+    def acquisition_controller(self):
+        return self._acquisition_controller
+
+    def on_acquisition(self, is_acquiring):
+        # Don't allow changes to acquisition/calibration ROIs during acquisition
+        if is_acquiring:
+            self._stream_controller.enable(False)
+            self._stream_controller.pause()
+        else:
+            self._stream_controller.resume()
+            self._stream_controller.enable(True)
+
+    def Show(self, show=True):
+        super(FastEMOverviewTab, self).Show(show)
+
+    @classmethod
+    def get_display_priority(cls, main_data):
+        # Tab is used only for FastEM
+        if main_data.role in ("mbsem",):
+            return 1
```

Comment:
Which tab should be shown by default? Overview? => use a bigger priority (eg, 2)